core/banking/snapshot.go

```diff
@@ -280,6 +321,22 @@ func (e *Engine) restoreScheduledTransfers(ctx context.Context, transfers []*che
 	return err
 }
 
+func (e *Engine) restoreBridgeState(ctx context.Context, state *types.BankingBridgeState, p *types.Payload) error {
+	var err error
```

Comment:
this `err` never seems to get set, why is it here?

Reply:
it is but at the end of the function. cleaning up just to make it more clear

Reply:
addressed